src/OrchardCore.Modules/OrchardCore.Workflows/Http/Controllers/HttpWorkflowController.cs

```diff
@@ -276,6 +276,11 @@ private IActionResult GetWorkflowActionResult()
                 return new EmptyResult();
             }
 
+            if (HttpContext.Items.TryGetValue(WorkflowHttpRedirectResult.Instance, out var value) && value is WorkflowHttpRedirectResult result)
```

Comment:
Is there an existing activity to do a redirect to a specific url? Then it could be used with the stored url.